cmd/osquery-perf/agent.go

```diff
@@ -280,6 +282,25 @@ func (a *agent) config() {
 		return
 	}
 
+	parsedResp := struct {
+		Packs map[string]struct {
+			Queries map[string]interface{} `json:"queries"`
+		} `json:"packs"`
+	}{}
+	if err := json.Unmarshal(res.Body(), &parsedResp); err != nil {
+		log.Println("json parse at config:", err)
+		return
+	}
+
+	var scheduledQueries []string
+	for packName, pack := range parsedResp.Packs {
+		for queryName := range pack.Queries {
+			scheduledQueries = append(scheduledQueries, packName+"_"+queryName)
+		}
+	}
+	a.scheduledQueries = scheduledQueries
+	log.Println("SCHEDULED QUERIES", scheduledQueries)
```

Comment:
Double checking you really want to log this.

Reply:
Nnnnope! Thank you :)